### PR TITLE
feat: Bump Axios to 0.28.0 and es5-ext to 0.10.63

### PR DESCRIPTION
**Pull request by rapotl**
Summary of changes 🔍
Build with Axios upgrades with recent Node 18 Migration; to further addresses the Comp Gov with Upgrade axios from 0.21.4 to 0.28.0 to fix the vulnerability.

**Testing** 🧪

- I ran the build locally and in the pipelines to make sure those worked.
           Hitting the localhost consuming local cardboard changes;      
image

- I ran storybook and it looks like all the styling stuff is still looking good.

**Checklist** ✔️

- [x]  Linked associated issue (if present)
- [x]  Added relevant labels
- [x]  Requested developer reviews
- [x]  Request UI review from design / PM
- [x]  Verify all code checks are passing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -176,7 +176,7 @@
     "overrides": {
         "minimist@1.2.5": "1.2.6",
         "follow-redirects": "1.15.6",
-        "es5-ext": "0.10.53"
+        "es5-ext": "0.10.63"
```
Comment (msnyder-msft) [inline package.json:179]: add this package to your PR name please so we know it got bumped

File: src/Adapters/ADTAdapter.ts [hunk 2]
```diff
@@ -583,7 +583,9 @@ export default class ADTAdapter implements IADTAdapter {
                     return null;
                 }
             });
-            return new ADTModelsData(axiosResult?.data);
+            return new ADTModelsData(
+                (axiosResult?.data as unknown) as IADTModel[]
```
Comment (msnyder-msft) [inline src/Adapters/ADTAdapter.ts:587]: why is this needed now?

File: src/Adapters/ADXAdapter.ts [hunk 3]
```diff
@@ -85,7 +85,7 @@ export default class ADXAdapter implements IADXAdapter {
                 const resultTimeSeriesData: Array<ADXTimeSeries> = []; // considering there is going to be multiple series to fetch data for
 
                 if (adxDataHistoryResults.data) {
-                    const primaryResultTables: Array<ADXTable> = adxDataHistoryResults.data.filter(
+                    const primaryResultTables: Array<ADXTable> = adxDataHistoryResults.data.data.filter(
```
Comment (msnyder-msft) [inline src/Adapters/ADXAdapter.ts:88]: hmmm why did the shape of the response change with the package update?

File: package.json (hunk 1 above)
Comment (rapotl) [reply package.json:179]: yep, updated that in PR name

File: src/Adapters/ADTAdapter.ts (hunk 2 above)
Comment (rapotl) [reply src/Adapters/ADTAdapter.ts:587]: there was a type clash in axiosResult; where its trying to assign values of DTModel[] to IADTModel[]

**Review by msnyder-msft — APPROVED** [review]

File: src/Adapters/ADTAdapter.ts (hunk 2 above)
Comment (msnyder-msft) [reply src/Adapters/ADTAdapter.ts:587]: I'm confused why this change is introducing that issue. Are you sure you don't need to go down another layer with data.data? That seemed to happen in the other places you were using axios results.

Comment (rapotl) [reply src/Adapters/ADTAdapter.ts:587]: layering doesn't seems to be an issue here! with different underlying shape, I checked the upstream and don't see any specific cast being made to axiosResult

File: src/Adapters/ADXAdapter.ts [hunk 4]
```diff
@@ -85,7 +85,7 @@ export default class ADXAdapter implements IADXAdapter {
                 const resultTimeSeriesData: Array<ADXTimeSeries> = []; // considering there is going to be multiple series to fetch data for
 
                 if (adxDataHistoryResults.data) {
```
Comment (pasanchMSFT) [inline src/Adapters/ADXAdapter.ts:87]: Shouldn't this change to **adxDataHistoryResults.data && adxDataHistoryResults.data.data**?

File: package-lock.json [hunk 5]
```diff
@@ -37337,6 +37274,17 @@
         "debug": "4.3.1",
         "openurl": "1.1.1",
         "yargs": "16.2.0"
+      },
+      "dependencies": {
+        "axios": {
```
Comment (pasanchMSFT) [inline package-lock.json:37279]: axios 0.21.4 is still included here

File: src/Adapters/BlobAdapter.ts [hunk 6]
```diff
@@ -175,7 +175,9 @@ export default class BlobAdapter implements IBlobAdapter {
                         headers: this.generateBlobHeaders(headers)
                     });
                     if (scenesBlob.data) {
```
Comment (pasanchMSFT) [inline src/Adapters/BlobAdapter.ts:177]: This should check for **scenesBlob.data && scenesBlob.data.data**

File: src/Adapters/ADXAdapter.ts (hunk 4 above)
Comment (rapotl) [reply src/Adapters/ADXAdapter.ts]: yes, I updated with additional checks on data.data

File: package-lock.json (hunk 5 above)
Comment (rapotl) [reply package-lock.json]: i updated the override block to update the pointed dependencies

File: src/Adapters/BlobAdapter.ts (hunk 6 above)
Comment (rapotl) [reply src/Adapters/BlobAdapter.ts]: done!

**Review by pasanchMSFT — APPROVED** [review]
Validated with Raj offline, updated methods worked correctly.
